doc/man1/flux-shell.rst

```diff
@@ -215,7 +215,37 @@ see the documentation for the specific plugin in question. The following
 options are supported by the builtin plugins of ``flux-shell``:
 
 **pty**
```

Comment:
Would it improve clarity if there were separate entries for:
- pty
- pty.interactive
- pty.ranks=OPT
- pty.capture

I'm not sure casual users will be thinking of how the shell options map to json objects in the jobspec, so presenting them as distinct options might make them more accessible?

Reply:
Ah, that is a good idea! I will make that change.